engine/src/main/java/com/arcadedb/schema/DocumentType.java

```diff
@@ -87,6 +87,8 @@ default Property createProperty(String propName, JSONObject prop) {
       p.setMandatory(prop.getBoolean("mandatory"));
     if (prop.has("notNull"))
       p.setNotNull(prop.getBoolean("notNull"));
+    if (prop.has("hidden"))
+      p.setNotNull(prop.getBoolean("hidden"));
```

Comment:
This should be `.setHidden()`, right?

Reply:
🤦 right, a classic copy/paste error. Fixed but still not working though...
Thanks for checking